### PR TITLE
[APIS-27] Clarification about inherited Education Organization Id values

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: api-guidelines/TIPS-FOR-SUCCESS.md
Comment (stephenfuqua) [inline api-guidelines/TIPS-FOR-SUCCESS.md:1]: This document will eventually be rewritten as a primer / cheat sheet that highlights easily overlooked normative guidance.

File: api-guidelines/v4.0/API-DESIGN-GUIDELINES/RESOURCES.md [hunk 1]
```diff
@@ -100,15 +101,63 @@ resource.
 }
 ```
 
+## Education Organizations
```
Comment (stephenfuqua) [inline api-guidelines/v4.0/API-DESIGN-GUIDELINES/RESOURCES.md:104]: This is the critical new section of the document. Maybe a bit too wordy?

**Review by stevenarnold-ed-fi — APPROVED** [review]
